package/skelcd-control-SLE_HPC.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Dec 11 16:56:07 UTC 2018 - dgonzalez@suse.com
+
+- Use FHS compliant path (fate#325482)
+- 15.0.10
```

Comment:
https://yastgithubio.readthedocs.io/en/latest/versioning/ strikes back. You should either target this PR to the SLE-15 branch or use the version number 15.1.0